Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -279,50 +299,19 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
           mGeneralOccupancy->SetBinError(xbin, ybin, 1e-15);
         }
       }
-    }
-  }
+      hAverageClusterSizeSummaryZPhi[iLayer]->Divide(hAverageClusterOccupancySummaryZPhi[iLayer]);
+      hAverageClusterOccupancySummaryZPhi[iLayer]->Scale(1. / mNRofs);
```

Comment:
This is correct but at the beginning of `monitorData` before starting to fill the histos again you have to add:
```
hAverageClusterSizeSummaryZPhi[iLayer]->Multiply(hAverageClusterOccupancySummaryZPhi[iLayer]);
hAverageClusterOccupancySummaryZPhi[iLayer]->Scale(mNRofs);
```
This is due to the fact that `monitorData` runs continuously on different TFs and hence, before refilling you have to remove the scaling factors. On the other side you would rescale too many times with these lines.

Reply:
OK, I did the same also for fine-binned histograms